docs/wooey_ui.rst

```diff
@@ -43,3 +43,27 @@ to parse this information), updates to the script version will result in a new
 version being created. If a command line library doesn't support versioning
 or the version has not been updated in a script, the Script Iteration counter
 will be incremented.
+
+Using URL parameters to pre-populate script parameters
+------------------------------------------------------
+
+Wooey supports pre-populating script parameters via URL parameters. This is
+useful for providing a link to a script with some parameters already set. The
+following rules apply.
+
+* URL parameters are specified as :code:`?parameter=value`
+* Multiple parameters may be specified by separating them with :code:`&`.
+  For example, :code:`?parameter1=value1&parameter2=value2`.
+* Parameters are specified by the name used in the script, **but in snake case**.
```

Comment:
This is not always true. A script can have multiple ways to refer to an argument, like `-f` and `--file` can be present. Unfortunately we only grab one of these (since it didn't matter at the time since we never exposed an api). So, there can still be some user confusion to what they should be parameterizing (suppose they pass f but its actually file that is needed). At some point, maybe we should have a little hover tooltip giving the right parameter to use as part of the UI.